src/models/StaticContent.js

```diff
@@ -0,0 +1,23 @@
+import mongoose from 'mongoose';
+import slugs from 'slugs';
+
+
+
+const Schema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  contentMD: {
+    type: String,
+    required: true,
+  },
+  contentHTML:{
+    type: String,
+    required: false,
```

Comment:
We should make this a required field. And then we would have a pre save function that converts the markdown to HTML so it's always there.